packages/sprotty/src/features/edge-layout/model.ts

```diff
@@ -61,11 +60,20 @@ export class EdgePlacement extends Object {
      * space between label and edge/connected nodes
      */
     offset: number;
+
+    /**
+     * where should the label be moved when move feature is enabled.
+     * 'edge' means the label is moved along the edge, 'free' means the label is moved freely, 'none' means the label is not moved.
+     * Default is 'edge'.
+     */
+    moveMode?: 'edge' | 'free' | 'none';
+
 }
```

Comment:
Shall we copy this interface to `sprotty-protocol` so it can be used on the server side, and mark this `@deprecated`?